ci/build.py

```diff
@@ -84,31 +85,46 @@ def is_mxnet_root(path: str) -> bool:
         curpath = parent
     return curpath
 
+def buildir() -> str:
+    return os.path.join(get_mxnet_root(), "build")
 
-def container_run(platform: str, docker_binary: str, command: List[str]) -> None:
+def container_run(platform: str, docker_binary: str, command: List[str], dry_run: bool = False) -> str:
```

Comment:
Can we infer the command from runtime_functions based on the platform?

Reply:
yes, it expects a function named build_<platform> as you can see in the code